### PR TITLE
vw java 11 compatibility

### PR DESCRIPTION
**Pull request by josepowera**
Fixes https://github.com/VowpalWabbit/vowpal_wabbit/issues/1610

### CODE REVIEW

File: java/pom.xml [hunk 1]
```diff
@@ -213,11 +213,11 @@
                     <nexusUrl>https://oss.sonatype.org/</nexusUrl>
                     <autoReleaseAfterClose>true</autoReleaseAfterClose>
                 </configuration>
-            </plugin>
+            </plugin>
```
Comment (jackgerrits) [inline java/pom.xml]: please remove trailing whitespace

**Comment by jackgerrits** [comment]
This needs to be up to date with master before CI will run. Also, does this break backwards compatibility with other versions of Java?

File: java/pom.xml (hunk 1 above)
Comment (jackgerrits) [inline java/pom.xml]:
```suggestion
             </plugin>
```

File: java/pom.xml [hunk 2]
```diff
@@ -226,7 +226,7 @@
                         </goals>
                     </execution>
                 </executions>
-            </plugin>
+            </plugin>
```
Comment (jackgerrits) [inline java/pom.xml]:
```suggestion
             </plugin>
```

**Comment by JohnLangford** [comment]
@josepowera can you update the pom.xml file?

**Comment by jackgerrits** [comment]
I really want to see this change in, can you move the changes from `pom.xml` to `pom.xml.in`?

**Comment by JohnLangford** [comment]
@deaktator @jon-morra-zefr any comments on this?  We are likely to merge some version of this shortly.

**Comment by jackgerrits** [comment]
@JohnLangford once CI passes this should be good to go.

**Comment by JohnLangford** [comment]
Merged, thanks :-)

**Pull request merged**